src/item.cpp

```diff
@@ -598,9 +604,6 @@ bool item::stacks_with( const item &rhs ) const
     if( corpse != nullptr && rhs.corpse != nullptr && corpse->id != rhs.corpse->id ) {
         return false;
     }
-    if( contents.size() != rhs.contents.size() ) {
-        return false;
-    }
```

Comment:
Why is this removed? `rhs.contents.end()` is not passed to `std::equal` below so this comparison is necessary.

Reply:
This was a mistake yea.